tests/git.zunit

```diff
@@ -3549,11 +3549,12 @@
         assert $5 same_as 'git fetch origin '
 
         run cat
-        assert ${#lines} equals 4
+        assert ${#lines} equals 5
         assert ${lines[1]} same_as "${fg[yellow]}another-branch$reset_color  ${fg[green]}branch$reset_color  "
```

Comment:
Out of curiosity, what are the whitespaces at the end of each line?

Reply:
The implementation of `_fzf_complete_tabularize` is bad, and so the whitespaces are inserted at the end when the number of the fields and the number of the its arguments are the same 😢

Reply:
Heck...